react/Wrapper.tsx

```diff
@@ -89,14 +90,22 @@ const Wrapper = withToast(function Wrapper(props: Props) {
 
   const skuItems = useMemo(
     () =>
+      props.skuItems ??
```

Comment:
How that makes it add more than one? Why we don't need to use `mapCatalogItemToCart`? Can you add a test case?

Reply:
because `skuItems` is an array of `CartItem`, which means that we already receive the prop in the correct format to add to the cart. but yes, I will do a test